core/src/scss/utilities/variables/components/_main-nav.scss

```diff
@@ -0,0 +1,151 @@
+@charset "UTF-8";
+
+//
+// $main-nav-colors
+//
+// Color map for the Main Nav component. Active color is also used to indicate "current" page.
+// The mobile colors apply also to desktop drop down menus.
+//
+// Markup: $main-nav-colors: (
+//    // Default Main Nav colors
+//    link:                                   $color-black,
+//    link--hover:                            $color-bright-red,
+//    border--hover:                          $color-bright-red,
+//    link--active:                           $color-black,
+//    border--active:                         $color-black,
+//    link--active-hover:                     $color-bright-red,
+//    border--active-hover:                   $color-bright-red,
+//    link--expanded:                         $color-black,
+//    border--expanded:                       $color-black,
+//    bg--mobile:                             $color-black,
+//    divider:                                $color-cool-grey,
+//    link--mobile:                           $color-white,
+//    link--mobile-hover:                     $color-white,
+//    border--mobile-hover:                   $color-white,
+//    link--mobile-active:                    $color-white,
+//    border--mobile-active:                  $color-vivid-red,
+//    link--mobile-expanded:                  $color-white,
+//    border--mobile-expanded:                transparent,
+//    border--mobile-active-expanded:         $color-white,
+//    light: (
```

Comment:
These need to be pulled out and reflected with the new multiple variable option.

Reply:
Ooops...meant to do that. I'll fix this and make the other changes. Thanks!